docs/to_clarify.md

```diff
@@ -4,30 +4,36 @@
+   a. Is the inability problem simply due to lack of integration across systems, and so the solution would "enhance back-end operational capabilities and business intelligence" by simply "integrating with existing solutions and databases"?
 
-    b. Does the solution contain capabilities to "create accurate guest profiles", or is that out of the scope?
+   b. Does the solution contain capabilities to "create accurate guest profiles", or is that out of the scope?
 
 ### Integration
 
 1. Stated under the section "Possible Use Cases" that the solution should "integrat(e) with existing solutions and databases".
 
-    a. What are the APIs (e.g. REST, SOAP, gRPC) used to communicate with these existing systems?
+   a. What are the APIs (e.g. REST, SOAP, gRPC) used to communicate with these existing systems?
 
-    b. Will documentation & assistance be provided if we face issues interfacing with these systems? For example, if we require additional API that is not yet implemented by an existing system, will we have to modify the existing codebase to implement the API that we need, or will someone be assisting us instead?
+   b. Will documentation & assistance be provided if we face issues interfacing with these systems? For example, if we require additional API that is not yet implemented by an existing system, will we have to modify the existing codebase to implement the API that we need, or will someone be assisting us instead?
 
-    c. On the same note, are there systems depending on the existing check-in/out system? If so, is the means of communication same as the answer to a.?
+   c. On the same note, are there systems depending on the existing check-in/out system? If so, is the means of communication same as the answer to a.?
 
 1. Stated under the section "Possible Use Cases" that the solution should "allow data transference capabilities, from room requests collected on the booking engine, to the check-in process till checkout". Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
 
 ### Security
 
 1. Stated under the section "Problem Statement" that the solution must be "safe and secured to protect guests' privacy". What exactly does this requirement mean, for example, are there certain security standards that the solution must adhere to?
 
+1. Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
+
```

Comment:
What does "this" refer to in this question? Also, correct me if I'm wrong, I think we can safely assume that the solution has to integrate with other platforms. I suggest we could remove this question if there are no objections.

Reply:
After reading the PDF that I have send earlier and Edeline's answer in the forum, I felt this question is redundant. Will remove if there is no disagreement.

Reply:
@Zhiyuan-Amos, do you have any objections to this?